### PR TITLE
[zh] Update security

### PR DESCRIPTION
**Pull request by my-git9**
[zh] Update security

### CODE REVIEW

File: content/zh/docs/reference/issues-security/security.md [hunk 1]
```diff
@@ -57,13 +52,14 @@ To make a report, please email the private [security@kubernetes.io](mailto:secur
 详细信息电子邮件到[security@kubernetes.io](mailto:security@kubernetes.io)列表。
 
 <!--
-You can also email the private [security@kubernetes.io](mailto:security@kubernetes.io) list with the security details and the details expected for [all Kubernetes bug reports](https://git.k8s.io/kubernetes/.github/ISSUE_TEMPLATE/bug-report.md).
+You can also email the private [security@kubernetes.io](mailto:security@kubernetes.io) list with the security details and the details expected for [all Kubernetes bug reports](https://github.com/kubernetes/kubernetes/blob/master/.github/ISSUE_TEMPLATE/bug-report.yaml).
+
 -->
-你还可以通过电子邮件向私有 [security@kubernetes.io](mailto:security@kubernetes.io) 列表发送电子邮件，邮件中应该包含[所有 Kubernetes 错误报告](https://git.k8s.io/kubernetes/.github/ISSUE_TEMPLATE/bug-report.md)所需的详细信息。
+你还可以通过电子邮件向私有 [security@kubernetes.io](mailto:security@kubernetes.io) 列表发送电子邮件，邮件中应该包含[所有 Kubernetes 错误报告](https://github.com/kubernetes/kubernetes/blob/master/.github/ISSUE_TEMPLATE/bug-report.yaml) 所需的详细信息。
```
Comment (tengqm) [inline content/zh/docs/reference/issues-security/security.md:58]: Wrap the long line properly?

File: content/zh/docs/reference/issues-security/security.md [hunk 2]
```diff
@@ -57,13 +52,14 @@ To make a report, please email the private [security@kubernetes.io](mailto:secur
...
 <!--
-You may encrypt your email to this list using the GPG keys of the [Product Security Committee members](https://git.k8s.io/security/README.md#product-security-committee-psc). Encryption using GPG is NOT required to make a disclosure.
+You may encrypt your email to this list using the GPG keys of the [Security Response Committee members](https://git.k8s.io/security/README.md#product-security-committee-psc). Encryption using GPG is NOT required to make a disclosure.
 -->
-你可以使用[产品安全团队成员](https://git.k8s.io/security/README.md#product-security-committee-psc) 
+你可以使用[安全响应委员会成员](https://git.k8s.io/security/README.md#product-security-committee-psc) 
 的 GPG 密钥加密你的电子邮件到此列表。使用 GPG 加密不需要公开。
```
Comment (tengqm) [inline content/zh/docs/reference/issues-security/security.md:63]:
```suggestion
你可以使用[安全响应委员会成员](https://git.k8s.io/security/README.md#product-security-committee-psc)的
GPG 密钥加密你的发往邮件列表的邮件。揭示问题时不需要使用 GPG 来加密。
```

File: content/zh/docs/reference/issues-security/security.md [hunk 3]
```diff
@@ -122,9 +118,9 @@ As the security issue moves from triage, to identified fix, to release planning
 ## 公开披露时间
 
 <!--
-A public disclosure date is negotiated by the Kubernetes product security team and the bug submitter. We prefer to fully disclose the bug as soon as possible once a user mitigation is available.
+A public disclosure date is negotiated by the Kubernetes Security Response Committee and the bug submitter. We prefer to fully disclose the bug as soon as possible once a user mitigation is available.
 -->
-公开披露日期由 Kubernetes 产品安全团队和 bug 提交者协商。我们倾向于在用户缓解措施可用时尽快完全披露该 bug。
+公开披露日期由 Kubernetes 安全响应委员会和 bug 提交者协商。我们倾向于在用户缓解措施可用时尽快完全披露该 bug。
```
Comment (tengqm) [inline content/zh/docs/reference/issues-security/security.md:123]:
```suggestion
公开披露日期由 Kubernetes 安全响应委员会和 bug 提交者协商。
我们倾向于在能够为用户提供缓解措施之后尽快完全披露该 bug。
```

File: content/zh/docs/reference/issues-security/security.md [hunk 4]
```diff
@@ -57,14 +52,18 @@ To make a report, please email the private [security@kubernetes.io](mailto:secur
 详细信息电子邮件到[security@kubernetes.io](mailto:security@kubernetes.io)列表。
 
 <!--
-You can also email the private [security@kubernetes.io](mailto:security@kubernetes.io) list with the security details and the details expected for [all Kubernetes bug reports](https://git.k8s.io/kubernetes/.github/ISSUE_TEMPLATE/bug-report.md).
+You can also email the private [security@kubernetes.io](mailto:security@kubernetes.io) list with the security details and the details expected for [all Kubernetes bug reports](https://github.com/kubernetes/kubernetes/blob/master/.github/ISSUE_TEMPLATE/bug-report.yaml).
+
 -->
-你还可以通过电子邮件向私有 [security@kubernetes.io](mailto:security@kubernetes.io) 列表发送电子邮件，邮件中应该包含[所有 Kubernetes 错误报告](https://git.k8s.io/kubernetes/.github/ISSUE_TEMPLATE/bug-report.md)所需的详细信息。
+你还可以通过电子邮件向私有 [security@kubernetes.io](mailto:security@kubernetes.io)
+列表发送电子邮件， 邮件中应该包含
```
Comment (ydFu) [inline content/zh/docs/reference/issues-security/security.md:59]:
```suggestion
列表发送电子邮件，邮件中应该包含
```

**Pull request merged**